sort-authors-list.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
```

Comment:
Yeah, this makes the script much simpler. And since it's not part of the official distribution, 👍